mapbox/services/uploads.py

```diff
@@ -45,15 +49,38 @@ def _get_credentials(self):
                 429: "Too many requests"})
         return resp
 
-    def stage(self, fileobj, creds=None, callback=None):
-        """Stages the user's file on S3
-        If creds are not provided, temporary credientials will be generated
-        Returns the URL to the staged resource.
+    def stage(self, fileobj, creds=None, callback=None, region_name=None):
+        """Stages data in a Mapbox-owned S3 bucket
+
+        If creds are not provided, temporary credentials will be
+        generated using the Mapbox API.
+
+        Note that when a s3:// URL is passed, identifying an object
+        in a bucket owned by the caller, the caller must ensure that
+        AWS credentials for access to that object are properly
+        configured.
+
+        Parameters
+        ----------
+        fileobj : file object or filename
+            A Python file object opened in binary mode or a filename.
+        creds : dict
+            AWS credentials allowing uploads to the destination bucket.
+        callback : func
+            A function that takes a number of bytes processed as its
+            sole argument.
+
+        Returns
+        -------
+        str
+            The URL of the staged data
         """
```

Comment:
👍  Thanks for reminding me about docstrings here, cut a ticket https://github.com/mapbox/mapbox-sdk-py/issues/195 to remind us to do the same for other service methods.